notebook/automl_classification.ipynb

```diff
@@ -875,7 +875,10 @@
     "        super().__init__(task, **config)\n",
     "\n",
     "        '''task=binary or multi for classification task'''\n",
-    "        if task in CLASSIFICATION:\n",
+    "        if isinstance(task, str):\n",
+    "            from flaml.automl.task.factory import task_factory\n",
+    "            task = task_factory(task)\n",
+    "        if task.is_classification():\n",
```

Comment:
Let's modify this notebook in a separate PR, and merge that PR at the time of new release. Otherwise, the notebook will become unusable before the new release.

Reply:
Hey! Is this still the case or has the release been cut now? I can spend some time this weekend trying to get this over the line

Reply:
I mean the notebook will be unusable between the merging of this PR and the release. To prevent that, we can add a flaml version number in the beginning of the notebook's "pip install". Remove other changes for the notebook in this PR and put them in a different PR. After we release the new version which contains this PR, we merge the other PR which modifies the notebook's content as well as the version number.

Reply:
Okay, sounds good. I'll get that done

Reply:
Reverted now